README.md

```diff
@@ -491,7 +491,9 @@ Excludes a subset of packages when running a command.
 $ lerna bootstrap --ignore component-*
 ```
 
-The `ignore` flag, when used with the `bootstrap` command, can also be set in `lerna.json` under the `bootstrapConfig` key. The command-line flag will take precendence over this option.
+The `ignore` flag, when used with the `bootstrap` and `exec` command, can also be set in `lerna.json` under the `bootstrapConfig` key. The command-line flag will take precendence over this option.
```

Comment:
this is a bit confusing, because you can't set the `ignore` option in `lerna.json` for the `exec` command. and this line mentions `bootstrapConfig`, which is for `bootstrap` only.